app/controllers/group_assignment_invitations_controller.rb

```diff
@@ -52,11 +52,19 @@ def authorize_group_access
     group_id = group_params[:id]
 
     return unless group_id.present?
+    group = Group.find(group_id)
+    check_group_members_not_exceed(group)
     return if group_assignment.grouping.groups.find_by(id: group_id)
 
     fail NotAuthorized, 'You are not permitted to select this team'
   end
 
+  def check_group_members_not_exceed(group)
```

Comment:
let's keep the naming consistent: `#validate_max_members_not_exceeded`

@tarebyte does it make sense to have `!` here?